browser/ui/views/side_panel/brave_side_panel.h

```diff
@@ -44,21 +53,30 @@ class BraveSidePanel : public views::View,
   // views::ResizeAreaDelegate:
   void OnResize(int resize_amount, bool done_resizing) override;
 
+  // views::View:
+  void ChildVisibilityChanged(View* child) override;
+  void OnThemeChanged() override;
+  gfx::Size GetMinimumSize() const override;
+  void AddedToWidget() override;
+
   void SetMinimumSidePanelContentsWidthForTesting(int width) {}
```

Comment:
I'm curious what this method is for.

Reply:
It was added for upstream compatibility.
As we replace whole upstream `SidePanel` impls with our `BraveSidePanel`, we should have same interfaces with it to prevent compile failure.